products/models.py

```diff
@@ -0,0 +1,134 @@
+from django.db import models
+
+
+class Product(models.Model):
+    name        = models.CharField(max_length=45)
+    dgree       = models.DecimalField()
+    ml          = models.IntegerField()
+    awards      = models.CharField(max_length=45)
+    created_at  = models.DateTimeField(auto_now_add=True)
+    updated_at  = models.DateTimeField(auto_now=True)
+    expire_date = models.CharField(max_length=45)
+    keep        = models.CharField(max_length=100)
+    grade       = models.DecimalField()
+    category    = models.ForeignKey("Category", on_delete=models.CASCADE, related_name="product")
+    brewery     = models.ForeignKey("Brewery", on_delete=models.CASCADE, related_name="product")
+    sidedish    = models.ManyToManyField("Sidedish", on_delete=models.CASCADE, related_name="product", null=True)
+    tag         = models.ManyToManyField("Tag", on_delete=models.CASCADE, related_name="product", null=True)
+
+    class Meta:
+        db_table = "products"
+
+    def __str__(self):
+        return self.name
+
+class Product_img(models.Model):
+    img_url = models.URLField()
+    product = models.ForeignKey("Product", on_delete=models.CASCADE, related_name="img")
```

Comment:
related_name은 복수로 만들어주세요~ related_name은 부모 인스턴스에서 역 참조하고 있는 자식 인스턴스 리스트를 가져올 때 사용하는 옵션이기 때문에 `복수`로 만들어주세요~

추가로 img -> `images`, 변수명은 줄여서 사용하지 않도록 주의해주세요~

```suggestion
    product = models.ForeignKey("Product", on_delete=models.CASCADE, related_name="images")
```